app/helpers/links_helper.rb

```diff
@@ -51,6 +51,14 @@ def mail_to_administrator
             body: permissions_help_email_body(current_user)
   end
 
+  def link_to_profile_terms
+    link_to t(:terms_and_conditions).downcase, user_terms_path, target: '_blank'
+  end
+
+  def link_to_forum_terms
+    link_to t(:forum_terms).downcase, discussions_terms_path, target: '_blank'
```

Comment:
Why downcasing them?

Reply:
Because if not it would be shown up like:
`Consultá las **Reglas del foro**` and I didn't like that.
That same translation is used for displaying the title